examples/assets-cloud/.env.example

```diff
@@ -0,0 +1 @@
+KEYSTONE_CLOUD_API_KEY=KEYSTONE_CLOUD_API_KEY
```

Comment:
Can we move the examples into the `examples-staging` directory for now? The `examples` directory is for our "blessed" examples that have been developed according to the details of `NEW-EXAMPLES.md`. Also, let's delete the `yarn.lock` files

Reply:
Okay!